<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>L</class>
 <widget class="QDialog" name="L">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>240</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>60</y>
     <width>237</width>
     <height>164</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <property name="leftMargin">
       <number>50</number>
      </property>
      <item>
       <widget class="QPushButton" name="LoginOKButton">
        <property name="text">
         <string>OK</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="LoginCancelButton">
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="0">
     <layout class="QFormLayout" name="formLayout_2">
      <item row="0" column="0">
       <layout class="QFormLayout" name="formLayout">
        <item row="0" column="0">
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>用户名</string>
              </property>
              <property name="buddy">
               <cstring>user_name_login</cstring>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="userNameLogin"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>密码  </string>
              </property>
              <property name="buddy">
               <cstring>user_password_login</cstring>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="userPasswordLogin"/>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="registerButton">
          <property name="text">
           <string>注册</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="1">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="0">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <zorder>registerButton</zorder>
  <zorder>label_4</zorder>
  <zorder>userNameLogin</zorder>
  <zorder>label_3</zorder>
  <zorder>userPasswordLogin</zorder>
  <zorder>verticalSpacer</zorder>
  <zorder>horizontalSpacer</zorder>
  <zorder>layoutWidget</zorder>
  <zorder></zorder>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>LoginCancelButton</sender>
   <signal>clicked()</signal>
   <receiver>L</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>262</x>
     <y>223</y>
    </hint>
    <hint type="destinationlabel">
     <x>275</x>
     <y>170</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
